### PR TITLE
Update Office 365 Connected Services Documentation

### PR DESCRIPTION
**Pull request by drielenr**
Update Office 365 Connected Services Documentation:
- Explain how to add Office 365 Connected Services in Visual Studio 2022
- Update the linked project to the newest version
- Update the instructions accordingly to the new project

### CODE REVIEW

**Comment by Jarbas-MSFT** [comment]
Thank you very much for your work on this PR, @drielenr. I reviewed your PR and applied some changes to the content. Please, let me know if you have any questions.

I noticed that your concept topic doesn't appear in the TOC (navigation). For better discoverability, we should add relevant topics to the TOC. What is your recommendation for a position in the TOC to surface your concept topic?

File: concepts/office-365-connected-services.md [hunk 1]
```diff
@@ -1,65 +1,65 @@
 ---
-title: "Call Microsoft 365 services in Visual Studio with the Microsoft Graph API"
-description: "This article describes how to configure Connected Services in Visual Studio to use Microsoft Graph in an ASP.NET MVC application that displays events from the currently signed in user's calendar."
+title: "Configure Microsoft 365 services with the Microsoft Graph API in Visual Studio"
+description: "This article describes how to configure connected services in Visual Studio to use Microsoft Graph in an ASP.NET MVC application that displays events from the currently signed in user's calendar."
```
Comment (John-Hart) [inline concepts/office-365-connected-services.md:3]: Connected Services is a feature name and should be capitalized, see https://learn.microsoft.com/en-us/visualstudio/azure/overview-connected-services?view=vs-2022

File: concepts/office-365-connected-services.md [hunk 2]
```diff
@@ -1,91 +1,121 @@
 ---
-title: "Call Microsoft 365 services in Visual Studio 2017 with the Microsoft Graph API"
-description: "You can use the Connected Services in Visual Studio to configure your app to call the Microsoft Graph API. This article describes how to get a signed in user's profile photo, upload it to OneDrive, and send an email with a sharing link to the photo."
+title: "Configure Microsoft 365 services with the Microsoft Graph API in Visual Studio"
+description: "This article describes how to configure connected services in Visual Studio to use Microsoft Graph in an ASP.NET MVC application that displays events from the currently signed in user's calendar."
```
Comment (John-Hart) [inline concepts/office-365-connected-services.md:3]:
```suggestion
description: "This article describes how to configure Connected Services in Visual Studio to use Microsoft Graph in an ASP.NET MVC application that displays events from the currently signed in user's calendar."
```

**Comment by John-Hart** [comment]
> Thank you very much for your work on this PR, @drielenr. I reviewed your PR and applied some changes to the content. Please, let me know if you have any questions.
> 
> I noticed that your concept topic doesn't appear in the TOC (navigation). For better discoverability, we should add relevant topics to the TOC. What is your recommendation for a position in the TOC to surface your concept topic?

How about adding it under the Develop section in the TOC

**Review by Lauragra — APPROVED** [review]

**Review by John-Hart — APPROVED** [review]

**Comment by John-Hart** [comment]
ready to merge

**Pull request merged**